userguide/storage.rst

```diff
@@ -611,22 +611,22 @@ Here are some examples:
    layout when extending the pool!
 
 
-.. _Detaching a Pool:
+.. _ExportDisconnect a Pool:
 
-Detaching a Pool
-~~~~~~~~~~~~~~~~
+Export/Disconnect a Pool
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 To export or destroy an existing pool, click the pool name,
 |ui-settings|, then
-:guilabel:`Detach`. The choice to retain or delete the contents of the
-pool depends upon the selections made in the screen shown in
-:numref:`Figure %s <zfs_detach_vol_fig>`.
+:guilabel:`Export/Disconnect`. The choice to retain or delete the
+contents of the pool depends upon the selections made in the screen
```

Comment:
"depends upon" is odd.  The choice doesn't depend on the options, the options must be set to do what the user chooses. Rewrite this sentence to be clearer.